### PR TITLE
Add library file for NATS

### PR DESCRIPTION
**Pull request by derekcollison**
This is for NATS messaging system, https://nats.io

This is the original: https://hub.docker.com/r/apcera/gnatsd/

/cc @wallyqs

### CODE REVIEW

**Comment by tianon** [comment]
LGTM :+1:

Nice and simple!

File: library/nats [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# maintainer: Derek Collison <derek@apcera.com> (@derekcollison)
+# maintainer: Waldemar Salinas <wally@apcera.com> (@)wallyqs)
```
Comment (psftw) [inline library/nats]: extra ``)`` in ``(@)wallyqs)``?

Comment (derekcollison) [inline library/nats]: Will fix..

**Comment by yosifkit** [comment]
LGTM, Build test of #961; e49936bba086d505d413b0d5268d77f1a42c32e5 (`nats`):

```console
$ url="https://raw.githubusercontent.com/docker-library/official-images/e49936bba086d505d413b0d5268d77f1a42c32e5/library/nats"
$ bashbrew build "$url"
Processing nats:0.6.4 ...
Processing nats:latest ...
$ bashbrew list --uniq "$url" | xargs test/run.sh
testing nats:0.6.4
	'utc' [1/3]...exec: "date": executable file not found in $PATH
Error response from daemon: Cannot start container 0d529990807f1fb2deefe0c125b994268485ef353fe5d83a0368a11e759fd20b: [8] System error: exec: "date": executable file not found in $PATH
failed
	'cve-2014--shellshock' [2/3]...passed
	'no-hard-coded-passwords' [3/3]...exec: "awk": executable file not found in $PATH
Error response from daemon: Cannot start container e5ee3636135dcdc2761ec4f70a4724d35d0c8d7a14e433535d77a19ebc9de1ad: [8] System error: exec: "awk": executable file not found in $PATH
failed
```

There is at least one failure in the above build log.
(failures are expected here :smile:)

**Comment by psftw** [comment]
More tiny static binary official images!  :+1:

**Pull request merged**